UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -38,8 +39,27 @@ public struct TypeAndStrength
 		[Tooltip("How much damage or heals If negative per 1u")]
 		public float EffectPerOne;
 	}
+	
+	#region Emotes
 
+	public List<EmoteTypeAndChance> EmoteEffects = new List<EmoteTypeAndChance>();
```

Comment:
might be worth pulling this out into its own class that inherits from BodyHealthEffect.cs
?

Reply:
Would I then have to make a override copy of `PossibleReaction()` that includes everything from BodyHealthEffects as well as the additional emote code? I'm not that familiar with virtual and override functions and if the code of these functions are still present in any of it's children.

Reply:
humm, depends if you want it still to apply BodyHealthEffects, BodyHealthEffects inherits from metabolism reaction, 

you can do base. Function name if you want the original functionality to be called
 
or just make it inherit from  metabolism reaction if you don't want it to apply BodyHealthEffects  stuff

Reply:
I'll probably call the BodyHealthEffects `PossibleReaction()` function in the new override the same way BodyHealthEffects calls the function from metabolism, want to keep BodyHealthEffects stuff in so people don't make two separate reactions for the same chemical. Thank you for help.

Reply:
humm, when would someone want health affect and laughter?

Reply:
It's not laughter specify, it just adds the tools to make any chemical have an emote side effect

Reply:
Like if you wanted to give a poison or narcotic an obvious tell

Reply:
code is good,  I'm just wondering Would it be better to make the action such as laughter, health affect and others effects? 
instead of reactions? 

humm, Only problem is you need to make 2 scriptable objects For one reaction

Reply:
p.s. this is for something for me to work on and just asking you if you think it would be good

Reply:
Just so it's easy for new reactions to be made and edited, I think it would be better to give the reactions more options that can simply be ignored, as opposed to making a new effect for the reaction you want to make. I hope I understood what you were asking correctly.